app/javascript/src/ChangePassword/index.jsx

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import React, {useState} from 'react'
+import { createReactWrapper } from 'utilities/createReactWrapper'
+import { LoginPage, Form, ActionGroup, Button } from '@patternfly/react-core'
+import { HiddenInputs, PasswordField, validateSingleField, FlashMessages } from 'LoginPage'
+import type {FlashMessage} from 'Types'
+
+import 'LoginPage/assets/styles/loginPage.scss'
+import brandImg from 'LoginPage/assets/images/3scale_Logo_Reverse.png'
+import PF4DownstreamBG from 'LoginPage/assets/images/PF4DownstreamBG.svg'
+
+type Props = {
+  lostPasswordToken: ?string,
+  url: string,
+  errors: (?FlashMessage)[]
+}
+
+const renderInputWithProps = (name, label, value, isValid, setValue, setIsValid, validateSingleField) => {
+  const props = {
+    isRequired: true,
+    name: `user[${name}]`,
+    fieldId: `user_${name}`,
+    label,
+    value,
+    isValid,
+    onChange: (value, event) => {
+      const isValid = validateSingleField(event)
+      setValue(value)
+      setIsValid(isValid)
+    }
+  }
+  return <PasswordField inputProps={props} />
+}
+
+const ChangePassword = ({lostPasswordToken, url, errors}: Props) => {
+  const [password, setPassword] = useState('')
+  const [isPasswordValid, setIsPasswordValid] = useState(undefined)
+  const [passwordConfirmation, setPasswordConfirmation] = useState('')
+  const [isPasswordConfirmationValid, setIsPasswordConfirmationValid] = useState(undefined)
+  const formDisabled = !isPasswordValid || !isPasswordConfirmationValid
+
+  return (
+    <LoginPage
+      brandImgSrc={brandImg}
+      brandImgAlt='Red Hat 3scale API Management'
+      backgroundImgSrc={PF4DownstreamBG}
+      backgroundImgAlt='Red Hat 3scale API Management'
+      loginTitle='Change Password'
+      footer={null}
+    >
+      {errors && <FlashMessages flashMessages={errors}/>}
+      <Form
+        action={url}
+        noValidate
+        id='edit_user_2'
+        acceptCharset='UTF-8'
+        method='post'
+      >
+        <input type='hidden' name='_method' value='put' />
+        <HiddenInputs />
+        {renderInputWithProps('password', 'Password', password, isPasswordValid, setPassword, setIsPasswordValid, validateSingleField)}
```

Comment:
IMHO using this function means the PasswordField is not well defined. Also using a function to render a component is an antipattern, since components are functions. Define a new component instead or let's improve PasswordField so that it's useful and doesn't need to redefine "wrapper" functions every time.